Sources/NSManagedObject+Sweetness.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  NSManagedObject+Sweetness.swift
+//  Tests
+//
+//  Created by Ellen Shapiro on 11/23/18.
+//
+
+import CoreData
```

Comment:
Initially, we discussed that the scope of this project was going to be limited by using `import Foundation`, if another import was needed it was out of scope. For that case we also made [SweetCoreData](https://github.com/UseSweet/SweetCoreData).